src/controller.py

```diff
@@ -270,7 +270,9 @@ def colorize_model(self):
         shutil.copyfile(s.slicing.stl_file, s.colorizer.copy_stl_file)
         save_splanes_to_file(self.model.splanes, s.slicing.splanes_file)
         call_command(s.colorizer.cmd)
+        lastMove = self.view.stlActor.lastMove
```

Comment:
why only last? Is it enough?

Reply:
This parameter was added for the task with centering. It is needed so that when the checkbox "set the model in the center" is unchecked, the model can return to its original position.